downstream/modules/platform/con-controller-restore-different-cluster.adoc

```diff
@@ -4,9 +4,15 @@
 
 = Restore to a different cluster
 
+[role="_abstract"]
 When restoring a backup to a separate instance or cluster, manual projects and custom settings under `/etc/tower` are retained. 
 Job output and job events are stored in the database, and therefore, not affected.
 
 The restore process does not alter instance groups present before the restore. 
 It does not introduce any new instance groups either. 
 Restored {ControllerName} resources that were associated to instance groups likely need to be reassigned to instance groups present on the new {ControllerName} cluster.
+
+[WARNING]
+====
+If you plan to use a backup and restore operation to migrate your {PlatformNameShort} instance to a different cluster or new set of hostnames, any {PlatformName} credentials you set up for {EDAcontroller} will break, causing your rulebook activations to fail. You must manually edit and update the {ControllerName} URL and associated credentials after the restore operation is complete to restore connectivity.
```

Comment:
Any reason this block of text is different to the other one? Not an issue, just curious why we didn't use the same text since its the same warning.